### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by prembala1710**

### CODE REVIEW

**Comment by prembala1710** [comment]
Testing

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# CliqInformerProject
```
Comment (prembala1710) [inline README.md:1]: Check Once

Comment (prem-bkp-test-github) [inline README.md:1]: Testing.

**Review by prem-bkp-test-github — CHANGES_REQUESTED** [review]
Testing

**Comment by prembala1710** [comment]
Look into it Once

**Comment by prembala1710** [comment]
Check Once More

**Comment by prembala1710** [comment]
Test

**Comment by prembala1710** [comment]
test

File: README.md (hunk 1 above)
Comment (prem-bkp-test-github) [inline README.md:1]: Provide Changes

**Review by prem-bkp-test-github — CHANGES_REQUESTED** [review]
Implement Changes

File: README.md (hunk 1 above)
Comment (prem-bkp-test-github) [reply README.md:1]: Test

**Comment by prembala1710** [comment]
Test

**Pull request closed without merging**